### DISCUSSION TITLE
Video tutorials to learn Zed

### DISCUSSION THREAD

**Discussion by quinncomendant** [zed-industries/zed#17028] [Feature Requests]
Judging by the many simple questions in the Zed Discord server, I think there are many people, like myself, struggling to learn how to get started using Zed. Anyone want to make some tutorial videos how to use it?

I'd like to learn things like:

- How to navigate the UI? (open files, switch tabs, hide/show drawers and panels, move tab to new window)
- How to manipulate text? (search/replace, filter, sort, format, move and duplicate lines, indent/outdent)
- What are the essential keybindings to learn?
- How do I modify a theme? How do I make highlighted text a different color?
- How do extensions work? Can I modify an extension?
- How to create auto-complete snippets?
- How to use a REPL?
- How to use AI?
- How to set up and run a task?
- How to replace selected text with the output from a shell command?

There are lots of videos reviewing Zed, but no good ones showing how to do basic stuff like this.

**Comment by MikeyBeez** [comment]
I couldn't find a good tutorial either.

**Comment by lykamspam** [comment]
the configuring is easy.
But how using AI? For example I can ask model but it not modyfy my files. No showing what I can change.

simple:
```
#include <stdio.h>

int main() {
 int a = 4.2;
 printf("%d\n", a);
return 0;
}
```

I ask: please make makefile, add  EXIT_SUCCESS not return 0;  How to do this? 
AI show me a diff and ask is ok or not, and make a executable file. (and get error)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
